pyramid_tm/__init__.py

```diff
@@ -1,15 +1,21 @@
 import sys
 import transaction
 
+from pyramid.httpexceptions import HTTPNotFound
+from pyramid.interfaces import IRequestExtensions
+from pyramid.interfaces import IRequestFactory
+from pyramid.request import Request, apply_request_extensions
 from pyramid.settings import asbool
-from pyramid.util import DottedNameResolver
+from pyramid.threadlocal import manager as request_manager
```

Comment:
Yes, placing the TM above exception views leverages `request.invoke_exception_view` to render exceptions that may occur after `EXCVIEW` tween but before pyramid_tm.